tests/cairo_native.rs

```diff
@@ -0,0 +1,843 @@
+#![cfg(not(feature = "cairo_1_tests"))]
+
+#[cfg(feature = "cairo-native")]
```

Comment:
I think you can set the conditional compilation for the whole file so you don't need to add `#[cfg(feature = "cairo-native")]` everywhere.

Reply:
You're right, fixed here [fc18c98](https://github.com/lambdaclass/starknet_in_rust/pull/943/commits/fc18c9813b136baffd5b8d5beda583284fcbe760)